requirements/requirements_trtllm.txt

```diff
@@ -0,0 +1,5 @@
+tiktoken
+jiwer
+tensorrt-llm==0.15.0.dev2024101500
+soundfile
```

Comment:
looks like you are not using soundfile as well

Reply:
![image](https://github.com/user-attachments/assets/7ae2bd52-3ed5-4ee4-8283-3f102d6e3c26)
It's required by datasets.